frontend/packages/pipelines-plugin/src/plugin.tsx

```diff
@@ -49,14 +28,7 @@ const plugin: Plugin<ConsumedExtensions> = [
   {
     type: 'ModelDefinition',
     properties: {
```

Comment:
We can move all the `ModelDefinition` entries to dynamic plugin as well.

Reply:
added the dynamic extension for model definition in the `console-extention.json` file but have kept this as is because the dynamic ex for this is not been consumed yer